app/application/views/header.php

```diff
@@ -14,9 +14,18 @@
         <script src="<?php echo base_url('/static/js/jquery.min.js'); ?>"></script>
         <script src="<?php echo base_url('/static/js/bootstrap.min.js'); ?>"></script>
         <script src="<?php echo base_url('/static/js/Chart.min.js'); ?>"></script>
-        
+        <script src="<?php echo base_url('/static/js/select2.min.js'); ?>"></script>
+
+
         <!-- App JS -->
         <script src="<?php echo base_url('/static/js/homepage.js'); ?>"></script>
+        <script type="text/javascript">
```

Comment:
Could you pop this block of JS into the `homepage.js` file referenced on the line above.